ledger/sandbox/src/main/scala/com/digitalasset/platform/index/StandaloneIndexServer.scala

```diff
@@ -119,16 +119,23 @@ class StandaloneIndexServer(
     }
   }
 
+  private def loadDamlPackages(): InMemoryPackageStore = {
+    // TODO is it sensible to have all the initial packages to be known since the epoch?
```

Comment:
It looks like the previous logic was to set `Instant.now` as the `knownSince` time. It probably makes sense to keep it as is.

Reply:
Interesting. The sandbox application always used `EPOCH` (see https://github.com/digital-asset/daml/pull/2091/files#diff-9ee24205d1c2b37348e86cc67c097e36L173), while the standalone index server used `Instant.now` before.

Why do we need to load any packages at all here? The read service contains ledger entries for package uploads, the package store should be updated from these entries.

If the initial packages are somehow packages that the ledger needs, but does not expose through the read service, then these should be uploaded at `EPOCH`, to make sure they predate any other ledger entries.

Reply:
it might be just a temporary solution. Let me change it to `Instant.now` so we can merge this.